docs/minimal/transformations.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Custom Transformation
+parent: Minimal downtime migrations
+nav_order: 4
+---
+
+# Minimal downtime migrations for MySQL
```

Comment:
Does the heading need to change?

Reply:
Created a different page and changed the heading